packages/webapp/tailwind.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 
                 'orange-light': '#FFD19A',
                 'orange-base': '#EC9455',
-                'orange-base-35': '#F5AE73',
+                'orange-base-35': '#f5ae7359',
```

Comment:
Is this one too many chars? And should be uppercase

Reply:
it's hex with transparency

Reply:
ah, alpha, thanks